### ISSUE TITLE
Units of Measure applied to literal bindings are ignored

### ISSUE THREAD

**Issue by reinux** [dotnet/fsharp#6929]
Instances of literal bindings assigned units of measure compile, but don't carry the measure.

**Code to reproduce**

```
type [<Measure>] rad
[<Literal>]
let pi = 3.14
let a = pi<rad>
let f (a:float<rad>) = a
f a
```

**Expected behavior**

Either an error on the assignment of a, or for f to accept pi<rad>

**Actual behavior**

```
Error	FS0001	This expression was expected to have type
    'float<rad>'    
but here has type
    'float'	F#
```

**Known workarounds**

`let a = pi*1.<rad>`

**Related information**

* Windows 10
* FSI 10.4.0 for F# 4.6
* Visual Studio 2019

**Comment by dsyme** [comment]
Yes the `<rad>` annotation in `pi<rad>` is being ignored when it should give an error

**Comment by abelbraaksma** [comment]
OR: @dsyme we could allow this, after all it makes sense to allow literal bindings to be augmented as measures just like literal values. Could add it here: https://github.com/fsharp/fslang-design/issues/496

**Comment by auduchinok** [comment]
@T-Gro The F# snippet in the bot response is not even valid F#. Why is this issue closed?

**Comment by T-Gro** [comment]
This issue is part of batch reproduced locally, but the previous bot indeed has problems with angle brackets in code samples - that is why its code boxes look like invalid. This is an issue for posting the comment, it is valid at repro time.

Will tackle it 👍

**Comment by dsyme** [comment]
> @T-Gro The F# snippet in the bot response is not even valid F#. Why is this issue closed?

Yeah, same problem with XML/HTML tag redaction being too over-zealous. 

It should be fixed by this: https://github.com/github/gh-aw/issues/22004

@T-Gro running 

```
gh aw upgrade
```

should fix, or else

```
gh extension upgrade gh-aw
gh aw compile
```
